src/app/page.tsx

```diff
@@ -1,51 +1,23 @@
-import { Button } from "@mantine/core";
-import QueryWrapper from "./query/wrapper";
+import { mockEmptyIssue, mockIssue, mockUser } from "@/mock/conversationMock";
+import HomePageCard from "../components/HomePage/HomePageCard";
+import type { Metadata } from "next";
 
+export const metadata: Metadata = {
+    title: "CommonGround",
+    description: "A platform for people to discuss and share their opinions.",
+};
+
+const issues = [mockIssue, mockEmptyIssue];
```

Comment:
In practice, issues will be handled by react-query with a server-prefetched response. So it would be more accurate if issues were defined in the `Page` component